.travis.yml

```diff
@@ -1,41 +1,46 @@
 language: node_js
 node_js:
-    - 13
-    - 12
-    - 10
+  - 13
+  - 12
+  - 10
 
```

Comment:
What was wrong with current indentation :)?

Reply:
acutally nothing, I think that looks just a little weird because of the `-` sign related to a removal in git. Actual indentation did not change :)